File: Merge Strings Alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string ans = "";
        int len;
        if (word1.length() < word2.length()) {
            len = word2.length();
        } else {
            len = word1.length();
        }
        for (int i = 0; i < len; i++) {
            if (i < word1.length())
                ans += word1[i];
            if (i < word2.length())
                ans += word2[i];
        }
        return ans;
    }
};

File: Leetcode-Problems/Merge Strings Alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        StringBuilder s = new StringBuilder();
        int len = Math.max(word1.length(), word2.length());
        for (int i = 0; i < len; i++) {
            if (i < word1.length())
                s.append(word1.charAt(i));
            if (i < word2.length())
                s.append(word2.charAt(i));
        }
        return s.toString();
    }
}